### ISSUE TITLE
Payment for suspended TKO subscription is unused

### ISSUE THREAD

**Issue by Sandplum** [TampereHacklab/mulysa#508]
### Payment for suspended TKO subscription is unused
<br>
When a member makes a payment with the TKO reference number while the TKO is suspended, the transaction is unused, the subscription remains Suspended, and the member and admin staff are unaware.
<br>
<br>
While suspended subscriptions should require an admin to authorize and restart the subscription, communication could be improved.
<br>
<br>
This is what the member can see on their User page (*redactions added by author) -->
<hr>`

> '![sus_tko_user_page_1](https://github.com/user-attachments/assets/247246eb-8845-48bd-a948-8f76a7e35981)

<hr>
<br>

<hr>

> `![sus_tko_user_page_2](https://github.com/user-attachments/assets/70770755-e5c0-4cb9-86fe-fea0ba91d2de)

<hr>
<br>
<br>

The Bank transactions page in Django admin shows the transaction is **unused**.
<hr>`

> '![banktrans](https://github.com/user-attachments/assets/0a1a5cab-b62d-4fd8-8384-eca5257f94d4)
<hr>

<br>
<hr>

> '![hasbeenused](https://github.com/user-attachments/assets/15f9e0c2-e186-42e5-bcd6-0acc69209dc9)
<hr>
<br>

**Comment by Sandplum** [comment]
### Possible solutions:

When a TKO subscription is in the Suspended state, and a new payment for that reference number is received, send an email to the member.  This email could be similar to the Door Access Denied email.  Also, create a log entry in the Users Log that both the member and the admin staff can read.  And add a comment to a new bank transaction that is unused due to Suspended TKO.

As a prevention measure, send an email to the member when a subscription changes from Active or Overdue, to Suspended.

**Comment by braaar** [comment]
I think at the very least, the user should be told in very clear terms what is going on and why their membership isn't active despite them having made a payment. A big red banner, like what I commented in https://github.com/TampereHacklab/mulysa/issues/511#issuecomment-2547622772 would be good.

**Comment by braaar** [comment]
I agree that email notification would be best here. People probably don't hop into mulysa to check stuff on their own initiative.
